app/controllers/exception_hunter/errors_controller.rb

```diff
@@ -20,5 +25,9 @@ def show
     def most_recent_errors
       Error.most_recent(params[:id])
     end
+
+    def dashboard_user
+      @dashboard_user ||= ExceptionHunter::Config.dashboard_user
```

Comment:
Same with this. Configuration is something internal, you never want it leaking out